src/main/resources/default/assets/javascript/multiLanguageField.js

```diff
@@ -175,7 +175,7 @@ MultiLanguageField.prototype.renderFlag = function (langCode) {
         const _globe = document.createElement('span');
         _globe.classList.add('mls-language-flag');
         _globe.classList.add('mls-language-globe');
-        _globe.textContent = String.fromCodePoint(127758);
+        _globe.innerHTML = '&#127757;';
```

Comment:
why not use an png like all the other icons use? or an icon font or an svg or what ever we use everywhere else?